README.md

```diff
@@ -16,6 +16,11 @@ Kite Connect is a set of REST-like APIs that expose many capabilities required t
 - [Python client documentation](https://kite.trade/docs/pykiteconnect/v4)
 - [Kite Connect HTTP API documentation](https://kite.trade/docs/connect/v3)
 
+## v4 - Breaking changes
+
+* Renamed ticker fields as per [kite connect doc](https://kite.trade/docs/connect/v3/websocket/#quote-packet-structure)
+* Renamed `bsecds` to `bcd` in `ticker.EXCHANGE_MAP`
```

Comment:
Just the last Saturday, I upgraded to Version: 3.9.5. and ran into problems.  Is this > Version: 3.9.5?